bin/trigger_azure_pipelines.py

```diff
@@ -123,7 +123,7 @@ def main() -> None:
     try:
         # ADO PAT (Azure DevOps Personal Access Token) with "Build" scope.

```

Comment:
Shall we update this comment?

Reply:
That's a good catch, Emily. I removed the comment as it is no longer relevant and var name is self-explanatory to represent ADO Token.